src/styles/index.scss

```diff
@@ -1,3 +1,145 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  background: grey;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  background: #fff;
+  width: 198px;
+  height: 406px;
+  top: 1px;
+  left: 1px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+}
+
+.card--image {
```

Comment:
One of the greatness of using SCSS instead of CSS, i the fact that you can use a concept called _Nesting_.
Nesting means that you can join class names to reduce code, and you could do it here. Foe example, for all _Elements_ that you have for a _Block_ you can use an "&" to join it.
Example:
```
.card {
  ...<card properties>
  
  &__image {
    ...<card__image properties>
  }

  &__text {
    ...<card__text properties>
  }
}
```
You could try doing so to all your classes that can, if you want